source/saml/tabs/rules.rst

```diff
@@ -0,0 +1,18 @@
+Rules for assigning authorisations
+----------------------------------
+
+
+It will be necessary to establish rules for assigning authorisations to your users (to give them a profile, for example).
+
+To do this, go to **Administration** > **Rules** > **Rules for assigning authorisations to a user**.

```

Comment:
**Rules for assigning authorisations to a user** is not implemented by the plugin. It uses its own rule set: **Rules** > ** Glpisaml - JIT import rules **

These rules can also be accessed via an additional header button in the plugins main configuration page.